### DISCUSSION TITLE
Why is my simulation results off?

### DISCUSSION THREAD

**Discussion by xshang93** [neperfepx/FEPX#89] [Q&A] [answered]
Hi @mkasemer , I'm running some simulation with FEPX on dual phase Ti64, with the parameters introduced in you paper (https://doi.org/10.1016/j.jmps.2017.03.013). However, I keep getting values a lot higher than the ones in your paper and other people's paper on Ti64. The yield strength at 0.1% offset I get is averaging 900-950 MPa, compared with the ones in your paper and other places, which are generally below 900 MPa.

For example:
This was a microstructure with about 90% alpha lamella, 10 prior beta grains, 1 alpha colony per grain, randomly assigned beta orientations and BOR enforced, which I think is comparable to the one in your paper with 40 grains (since my domain is only 1x1x1), 1 alpha colony per grain, without beta lamella.

![1_10_1_6_geom](https://user-images.githubusercontent.com/86078353/173096918-de2799d9-3e15-4655-aadc-004f6029ab80.png)

![1_stress_strain](https://user-images.githubusercontent.com/86078353/173096403-b7aa123e-6532-4f70-90b9-5ae984404670.png)
 
As you can see, the 1% yield strength is at 950 MPa, more than 100 MPa higher than the result in your paper (840 MPa I believe, see below).

![image](https://user-images.githubusercontent.com/86078353/173097373-b285f0e5-11df-40a6-a88b-d75d1003ef2b.png)

So what is wrong with my result? I have done mesh convergence study and the result converged.
My `simulation.config` file is attached for your reference.
[simulation.txt](https://github.com/neperfepx/FEPX/files/8880113/simulation.txt)

Thank you

**Comment by mkasemer** [comment, marked as answer]
First, you have so few grains / colonies per grain that it is hard to compare statistics. You could get a different result every time.

Second, the prior beta grains are not randomly oriented in my paper. Please review the orientation distribution functions in the JMPS paper.

Finally, a quick glance at your simulation file indicates that you are not using the same parameters used in that paper - namely I note issues with your slip strengths for the hexagonal phase, though you might want to check all parameters if you intend for a 1:1 comparison.

**Reply by xshang93** [reply]
> First, you have so few grains / colonies per grain that it is hard to compare statistics. You could get a different result every time.
> 
Thanks for the quick response. The grains are few for this case, but I have done 25 grains/domain and the results are even higher. And although the results for the 10 grain case fluctuate a little bit, the results are still quite high each time.
> Second, the prior beta grains are not randomly oriented in my paper. Please review the orientation distribution functions in the JMPS paper.
> 
That's a validate point. But to my knowledge shouldn't randomly arranged orientations lead to a lower yield strength, compared with a textured one? Contrary to what I'm getting.
> Finally, a quick glance at your simulation file indicates that you are not using the same parameters used in that paper - namely I note issues with your slip strengths for the hexagonal phase, though you might want to check all parameters if you intend for a 1:1 comparison.
>
Right, I accidentally used the slip strength values in the fepx examples, I will correct that and see how it goes.

**Reply by mkasemer** [reply]
@xshang93: As a note, the material parameters used in the JMPS paper were interim values, which were later refined to what are currently in the FEPX examples. I would use the latter in any eventual study.

Regarding strength, two points to note:
- Regarding the strength of beta texture: Assume the microstructure was single phase beta. As texture becomes stronger, the overall stress-strain behavior of the material will approach that of the single crystal. However, it's impossible to say what that behavior will be unless you know the orientation of the single crystal: the yield behavior of a single crystal could be lower or higher than that of the polycrystal with randomly oriented grains, it is entirely dependent on the orientation of that single crystal.
- Keep in mind, too, that this is a _transformation_ structure. The majority of the volume is alpha grains, whose orientations are only _related_ to the prior beta grain orientations. Again, consider the case of a perfect single crystal of beta with no rotation. While this beta grain has the potential to have a high strength, the 12 variants of alpha that form within the beta grain may vary in directional strength.

So overall, while your intuition is not baseless, you have to keep these points in mind when attempting to rectify your understanding of the behavior with your observations.

**Reply by xshang93** [reply]
That makes sense, the relations are indeed not monotonic. Thank you for the explanation!
